docs/api/README.md

```diff
@@ -257,3 +257,13 @@ window.addEventListener("keplr_keystorechange", () => {
     console.log("Key store in Keplr is changed. You may need to refetch the account info.")
 })
 ```
+
+### Change Chain Event
+
+Similarly, when the user switches the network from the extension menu, the webpage might need to be made aware of the currently selected chain. Keplr emits a `keplr_chainchange` event to the webpage's window each time the user selects a new chain from the extension. You can request the currently selected chain based on this event listener.
```

Comment:
Just confirming, is this event only for requesting `new` selected chain (peeking at other event handlers, that's what they do), or can we also get the current one (as in, without chain change user action)?
In other words, will it be fired a first time, on the initial chain the user is on?

Either way, the documentation entry will need to reflect that it's also/only (depending on your answer to that) for **new** selected chain, as opposed to the `current` terminology in this diff